### DISCUSSION TITLE
🐛Bug - Previously invited SSO users must be invited using the same case

### DISCUSSION THREAD

**Discussion by leahy268** [tinacms/tinacms#3926] [Help]
Hi Guys,

This appears to be a bug.  When inviting users to our organization we regularly have issues with them not being able to accept the invite.  Instead Tina asks them to sign up and they never see our organization.  The invite also always appears pending.

Recently I deleted a couple of these users and then reinvited them and got this error message:
![image](https://github.com/tinacms/tinacms/assets/58454534/f0b559b0-9efa-4943-a6af-9aa0528b13aa)
**Figure: UserNotFoundException: User does not exist**

This error was deemed to be an uppercase/lowercase issue.  I initially invited the user as firstnamelastname@email and then had to change that to FirstnameLastname@email to match their UPN from when they signed up with the invite.

This is weird behavior, as if I delete the user from the organization then I must re-invite them so 'user does not exist' shouldn't be an error that should be expected.  But also email addresses are not case sensitive.

Thanks

Warwick

**Comment by adamcogan** [comment]
This is a blocking issue! 
Upper/lower case issues are always painful. ⚠️

This is a discussion. Should this be filled as an 'issue'?
-a
www.adamcogan.com

**Comment by leahy268** [comment]
>This is a discussion. Should this be filled as an 'issue'?
Done - Moved to an issue
https://github.com/tinacms/tinacms/issues/3937

**Comment by jamespohalloran** [comment]
Closing this discussion in favour of [the issue](https://github.com/tinacms/tinacms/issues/3937) to keep conversation in one place.
